mixserv/src/main/java/hivemall/mix/server/MixServerHandler.java

```diff
@@ -133,22 +146,21 @@ private void mix(final ChannelHandlerContext ctx, final MixMessage requestMsg, f
                 short diffClock = partial.diffClock(localClock);
                 partial.add(weight, covar, deltaUpdates, scale);
 
-                if(diffClock >= syncThreshold) {// sync model if clock DIFF is above threshold
+                if(diffClock >= syncThreshold) { // sync model if clock DIFF is above threshold
                     float averagedWeight = partial.getWeight(scale);
                     float meanCovar = partial.getCovariance(scale);
                     short globalClock = partial.getClock();
-                    responseMsg = new MixMessage(event, feature, averagedWeight, meanCovar, globalClock, 0 /* deltaUpdates */);
+                    responseMsg = new MixMessage(
+                            event, feature, averagedWeight, meanCovar, globalClock,
+                            0 /* deltaUpdates */);
+                    ctx.writeAndFlush(responseMsg);
```

Comment:
This is my bad for `writeAndFlush`, so I'll fix it.

Why `SessionObject session = getSession(msg)` and `PartialResult partial = getPartialResult(msg, session)` is outside `mix`? If these sentences move to `mix`, argument `session` is not necessary anymore.